api/cloud/oc_cloud_manager.c

```diff
@@ -665,11 +697,17 @@ cloud_manager_login_async(void *data)
   return OC_EVENT_DONE;
 
 retry:
+  if (!cloud_schedule_retry(ctx, OC_CLOUD_ACTION_LOGIN,
+                            cloud_manager_login_async,
+                            OC_STATUS_SERVICE_UNAVAILABLE)) {
```

Comment:
same question as in register handler, is the `OC_STATUS_SERVICE_UNAVAILABLE` correct here?

Reply:
removed 3 parameter